arangod/Cluster/AgencyCache.cpp

```diff
@@ -286,26 +284,12 @@ void AgencyCache::run() {
       }
       LOG_TOPIC("afede", TRACE, Logger::CLUSTER)
           << "AgencyCache: poll polls: waiting for commitIndex " << commitIndex;
-      return AsyncAgencyComm().poll(60s, commitIndex);
+      // This is intentionally 61s timeout to avoid a client timeout, since
+      // the server returns after 60s by default. This avoids broken
+      // connections.
+      return AsyncAgencyComm().poll(61s, commitIndex);
     };
 
-  // the following variables are recycled for each run, so that we can save
-  // a few memory allocations in the lifetime of the process
-  
-  // ids of callbacks to call
-  std::vector<uint64_t> toCall;
-
-  // we pass the uniq variable to handleCallbacksNoLocks, but we don't
-  // need it ourselves
-  std::unordered_set<uint64_t> uniq;
-
-  // Plan changes
-  std::unordered_set<std::string> pc;  
-  
-  // Current changes
-  std::unordered_set<std::string> cc;  
```

Comment:
Ok, these unordered_sets were previously reused for every round, making it possible to save memory allocations.
This optimization has now been removed. Was this intentional?